src/lib/screen-utils.js

```diff
@@ -10,8 +10,13 @@ import layout, {STAGE_DISPLAY_SCALES, STAGE_SIZE_MODES, STAGE_DISPLAY_SIZES} fro
  */
 
 const STAGE_DIMENSION_DEFAULTS = {
-    spacingBorderAdjustment: 9,
-    menuHeightAdjustment: 40
+    // referencing css/units.css,
+    // spacingBorderAdjustment = 2 * $full-screen-top-bottom-margin +
+    //   2 * $full-screen-border-width
+    fullScreenSpacingBorderAdjustment: 12,
```

Comment:
Note that this 12px includes both borders, the 3px padding above the stage's top border, and (this one might be debatable) the 3px padding below the stage's bottom border.

Reply:
I also like the little bit of bottom padding in the browser, but it may not be ideal in a mobile situation (e.g. if it adds scrollbars).